test/GovUserKeeper.test.js

```diff
@@ -103,6 +151,279 @@ describe("GovUserKeeper", () => {
           ["6", "9"]
         );
       });
+
+      it("should revert if nft is not supported", async () => {
```

Comment:
So this test should be written inside a different describe. This is "Plain GovUserKeeper". Please create a new describe with a "No ERC20 GovUserKeeper" and write this test there